contracts/dnsregistrar/DNSRegistrar.sol

```diff
@@ -82,12 +82,12 @@ contract DNSRegistrar is IDNSRegistrar {
      * @param proof The proof record for the first element in input.
      */
     function proveAndClaim(bytes memory name, DNSSEC.RRSetWithSignature[] memory input, bytes memory proof) public override {
-        proof = oracle.submitRRSets(input, proof);
+        // proof = oracle.submitRRSets(input, proof);
```

Comment:
Does `proveAndClaim` still do any sort of proving (if it tries to prove, I am guessing it will require lots of gas). I am not so sure how DNSRegistrar work alongside with the stateless DNSSEC Oracle by reading this PR

Reply:
This is still a WIP - I haven't implemented the registrar changes yet.